src/Raven.Server/Documents/Operations/Operations.cs

```diff
@@ -377,8 +377,8 @@ public enum OperationType
             [Description("Collection import from csv")]
             CollectionImportFromCsv,
 
-            [Description("Database migration")]
-            DatabaseMigration,
+            [Description("RavenDB Database migration")]
+            DatabaseMigrationRavenDb,
```

Comment:
`DatabaseMigrationFromRavenDB`

Not sure what implications does this rename have? Backward compatibility needed?

Reply:
Old migrations from RavenDB will display only the documents section (studio only)